src/Diagnostics/atmos_les_default.jl

```diff
@@ -404,10 +489,21 @@ function atmos_les_default_collect(dgngrp::DiagnosticsGroup, currtime)
 
         # FIXME properly
         if isa(bl.moisture, EquilMoist)
-            if !iszero(thermo.moisture.q_liq)
+            if !iszero(thermo.moisture.q_liq + thermo.moisture.q_ice)
```

Comment:
Can we call `has_condensate(::ThermodynamicState)` here?

Reply:
done